### PR TITLE
move updated sojourner into main section

### PR DESCRIPTION
**Pull request by joel-lbth**
this neat pwa has been updated with the 2023 schedule, it includes the ability to watch streams and has a nice event map, jarek will be at the conference too, so please give him a boost!

### CODE REVIEW

File: content/schedule/mobile.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 <h3>Apps for Mobile Devices, tested for 2022</h3>
 <ul>
+<li><b>Progressive Web App: Sojourner</b><br />
+    <a href="https://github.com/loomchild/sojourner-web">Source code</a>,<br />
+    Available <a href="https://fosdem.sojourner.rocks">here</a>.<br />
+</li>
```
Comment (loomchild) [inline content/schedule/mobile.html:15]: Actually maybe better link directly to this year's app: `https://2023.fosdem.sojourner.rocks`

**Comment by johanvdw** [comment]
Thanks for the update.

A remark: https://2023.fosdem.sojourner.rocks/type/Developer%20rooms
and https://2023.fosdem.sojourner.rocks/type/Developer%20rooms

If two rooms are used for a track, they don't show up. Apart from that, it might be a good idea to make it more clear that the sessions in D.* are only happening online. (Something we are working on on the fosdem website as well.)

**Comment by loomchild** [comment]
Thank you, I will look into this over the weekend.

**Comment by loomchild** [comment]
@johanvdw Thank you very much for your help. I have fixed a few things (showing Keynotes correctly, marked D.* rooms as online).

I don't understand your remark above, these look like two identical URLs. Could you please clarify?

I also don't understand what's the issue if two rooms are used for the same track. Where they don't show up? Are both rooms used on the same day (I've seen tracks with different rooms for Saturday and Sunday and it seems to work OK).  Could you share an example illustrating the issue?

Perhaps it'd be best if you create issues directly in project repo: https://github.com/loomchild/sojourner-web ?

**Comment by loomchild** [comment]
Ah, I see that some tracks are using two rooms on the same day. I will fix this one.

**Pull request merged**